src/docs/product/issues/issue-details/error-issues/index.mdx

```diff
@@ -51,6 +51,10 @@ A _suspect commit_ is a commit that's been identified as potentially having caus
 
 The tags displayed in the main section of this page are specific to the event that you're viewing. In contrast, the tags displayed in the right-hand sidebar are a summary of all tag values for all events included in the issue. You can set your own tags to make them more useful for debugging as described in <PlatformLink to="/enriching-events/tags/">Customize Tags</PlatformLink>.
 
+## Screenshots
+
+Sentry provides the ability to take a screenshot and include it as an attachment when a user experiences an error. Screenshot attachments sent as a part of an event are displayed in this section. This feature only applies to SDKs with a user interface.
```

Comment:
Since we're not dealing with a space issue here, I think it makes sense to add the whole list. 
```suggestion
Sentry provides the ability to take a screenshot and include it as an attachment when a user experiences an error. Screenshot attachments sent as a part of an event are displayed in this section. This feature only applies to SDKs with a user interface. It's supported for the following SDKs:

- [.NET Xamarin](/platforms/dotnet/guides/xamarin/)
- [Android](/platforms/android/enriching-events/screenshots/)
- [Flutter](/platforms/flutter/enriching-events/screenshots/)
- [iOS](/platforms/apple/guides/ios/enriching-events/screenshots/)
- [JavaScript Electron](/platforms/javascript/guides/electron/enriching-events/screenshots/)
- [Unity](/platforms/unity/enriching-events/screenshots/)

```